### PR TITLE
Add Crypto Market Price to each Post

### LINKED ISSUE #1201: Add crypto market price on single post
We can show market price on single post when a post is using a tag related to a crypto symbol: for example when a post has these tags: `bitcoin`, `eth`, `trading`, `eos` we would show on right sidebar of the single post page the block "Market" with price of BitCoin Ethereum and EOS.

See:
image

### PR DESCRIPTION
**Pull request by jm90m**
Fixes #1201 

Changes:
- Make CryptoTrendingCharts component take an array of tags to render
- Remove SteemTrendingCharts component
- Add CryptoTrendingCharts to posts with crypto tags

### CODE REVIEW

File: src/client/components/Sidebar/CryptoChart.js [hunk 1]
```diff
@@ -61,18 +69,27 @@ class CryptoChart extends React.Component {
       loading: true,
     });
     fetchCryptoPriceHistory(symbol).then((response) => {
```
Comment (Sekhmet) [inline src/client/components/Sidebar/CryptoChart.js]: We should handle catch as well to prevent unhandled promise error.

Comment (jm90m) [reply src/client/components/Sidebar/CryptoChart.js]: good point will add

**Comment by jm90m** [comment]
@Sekhmet added catch block

**Comment by Sekhmet** [comment]
I feel like the way we use for refreshing is a little bit too much hacky.
Personally, I would move chart data to redux store. I would create an action `loadPrices(SYM)` and dispatch it for all needed currencies on `handleOnClickRefresh`.
I would create `CryptoTrendingChartsContainer` that adds `loadPrices action`, and `CryptoChartContainer` that subscribes to its state in redux.

**Comment by jm90m** [comment]
@Sekhmet I guess moving it to the redux store would be alot better, and passing it through to reducers probably in the `appReducer` where we store rate there.

**Pull request merged**